spec/system/admin/order_cycles/simple_spec.rb

```diff
@@ -781,14 +781,14 @@
   def expect_payment_methods_to_be_checked_for(distributor)
     distributor.distributor_payment_method_ids.each do |distributor_payment_method_id|
       expect(page).to have_checked_field
-        "order_cycle_selected_distributor_payment_method_ids_#{distributor_payment_method_id}"
+      "order_cycle_selected_distributor_payment_method_ids_#{distributor_payment_method_id}"
```

Comment:
I confirmed for this spec that the string was ignored and not passed on as an argument. Adding parenthesis fixes the code. I will add a commit.